contracts/BondVault.sol

```diff
@@ -81,52 +81,59 @@ function _DAO() internal view returns(iDAO) {
         mapAddress_listedAssets[asset].bondedLP[member] = memberDetails.bondedLP;
         mapAddress_listedAssets[asset].claimRate[member] = memberDetails.claimRate;
         mapAddress_listedAssets[asset].lastBlockTime[member] = memberDetails.lastBlockTime;
         return true;
     }
+    function migrateBondedAssets(address asset, address newBond) external onlyDAO returns (bool){
+       uint256 assetBal = iBEP20(asset).balanceOf(address(this));
+       iBEP20(asset).transfer(newBond, assetBal);
+        return true;
+    }
 
-    function depForMember(address asset, address member, uint LPS) public onlyDAO returns(bool){
+    function depForMember(address asset, address member, uint LPS) external onlyDAO returns(bool){
          if(!mapAddress_listedAssets[asset].isMember[member]){
           mapAddress_listedAssets[asset].isMember[member] = true;
           arrayMembers.push(member);
           mapAddress_listedAssets[asset].members.push(member);
         }
-        if(mapAddress_listedAssets[asset].bondedLP[member] > 0){
+        if(mapAddress_listedAssets[asset].bondedLP[member] != 0){
             cFMember(asset, member);
         }
+
         mapAddress_listedAssets[asset].bondedLP[member] += LPS;
         mapAddress_listedAssets[asset].lastBlockTime[member] = block.timestamp;
-        mapAddress_listedAssets[asset].claimRate[member] = mapAddress_listedAssets[asset].bondedLP[msg.sender]/(iBOND(_DAO().BOND()).bondingPeriodSeconds());
+        mapAddress_listedAssets[asset].claimRate[member] = mapAddress_listedAssets[asset].bondedLP[member] / iBOND(_DAO().BOND()).bondingPeriodSeconds();
         return true;
     }
-    function depINIT(address asset, address member, uint LPS) public onlyDAO returns (bool){
+    function depINIT(address asset, address member, uint LPS) external onlyDAO returns (bool){
          if(!mapAddress_listedAssets[asset].isMember[member]){
           mapAddress_listedAssets[asset].isMember[member] = true;
           arrayMembers.push(member);
           mapAddress_listedAssets[asset].members.push(member);
         }
         mapAddress_listedAssets[asset].bondedLP[member] += LPS;
         mapAddress_listedAssets[asset].lastBlockTime[member] = block.timestamp;
-        mapAddress_listedAssets[asset].claimRate[member] = mapAddress_listedAssets[asset].bondedLP[member]/(20736000);//must be changed for mainet
+        mapAddress_listedAssets[asset].claimRate[member] = mapAddress_listedAssets[asset].bondedLP[member] / 20736000;//must be changed for mainet
         return true;
     }
 
-     function cBLP(address member, address asset) public onlyDAO returns (uint){
-        uint256 _secondsSinceClaim = block.timestamp - (mapAddress_listedAssets[asset].lastBlockTime[member]); // Get time since last claim
-        uint256 rate = mapAddress_listedAssets[asset].claimRate[member];
-        uint claimAmount;
+     function cBLP(address member, address asset) public onlyDAO returns (uint claimAmount){
```

Comment:
Add check for isMember.
Add claimAmount named variable to the return.
Refactor _secondsSinceClaim